defaults/main.yml

```diff
@@ -4,7 +4,7 @@
 monasca_checks: {}
 monasca_agent_system_only: False
 agent_reconfigure_script: /usr/local/bin/monasca-reconfigure
-
+default_monasca_agent_git_repo: https://git.openstack.org/openstack/monasca-agent
```

Comment:
I think you can just call this ```monasca_agent_git_repo```, set the default and then override the variable if you need to. No need for two variables.

Reply:
Is this better?

Reply:
It works, although it's more canonical to put the default in the defaults/main file